services/services.go

```diff
@@ -83,6 +83,9 @@ func Init() {
 	ready.Add(1)
 	go latestExportedStatisticDayUpdater(ready)
 
+	ready.Add(1)
+	go stripeEmailUpdater(ready)
```

Comment:
The issue by placing it here is, that all of our frontend instances will run this check. So 4 mainnet instances and another one for each testnet instance we have.  
I'm note sure if we have a service besides the notification sender that runs only once though. Probably best to talk with StefanF what the best approach here is. Maybe introduce a new config flag that is only set on one instance, or a microservice approach of adding a new service for such things (we could combine it with the notification sender server and rename it to user service or something of that kind).

Reply:
created user service for this